skt/reporter.py

```diff
@@ -414,11 +414,7 @@ def getjobresults(self):
         return result
 
     def getreport(self):
-        msg = list()
-
-        if self.cfg.get("krelease"):
-            msg.append("result report for kernel %s"
-                       % self.cfg.get("krelease"))
+        msg = ['Hi,\nthanks for your contribution to Linux kernel!\n']
```

Comment:
"Hello" works for me, however since we will be running skt internally too I wanted the message to work with that as well and your version seems too much for that. Maybe we can think of something in the middle?

Reply:
Are you talking about "Hello", or the whole introduction? I'm not sure what to replace "Hello" with, which would be similarly respectful. Regarding the paragraph, I think it's OK to mention our team and company names internally, we want to be known and recognized inside too :)

Could you give me suggestions or specific points you'd like changed?

Reply:
"Hello" is fine. The rest of the introduction looks too formal, robotic and not very positive. Especially with combination of failed tests it sounds like "hey, you suck and here is a robot to call you out". From my experience with some upstream communities, I think it's very important to start positive.

Also, another point regarding mentioning the company and team name: since this project is in the wild now and anybody - including other people without any involvement with us - can use it, having it hardcoded (without any possibility to configure) would just cause confusion. We might want to add a configurable signature, but that's for another day.